Buildings/Fluid/FixedResistances/PressureDrop.mo

```diff
@@ -10,7 +10,10 @@ model PressureDrop
                   Dialog(group = "Transition to laminar",
                          enable = not linearized));
 
-  final parameter Real k = if computeFlowResistance then
+  // Below is a temporary fix for Dymola 2020x beta2. For release, replace
+  // if (dp_nominal_pos > Modelica.Constants.eps) then with
+  // if computeFlowResistance then
+  final parameter Real k = if (dp_nominal_pos > Modelica.Constants.eps) then
```

Comment:
Should keep the `if (dp_nominal_pos > Modelica.Constants.eps) then`, or change back to `if computeFlowResistance then` for merging to master?

Reply:
Thanks, this should use `computeFlowResistance` which I fixed in https://github.com/lbl-srg/modelica-buildings/pull/2212/commits/2a603c2afa94e97f27f82ca4569f1c402c05ff93